src/gui/torrentmodel.h

```diff
@@ -85,12 +83,12 @@ class TorrentModel : public QAbstractListModel
 
     explicit TorrentModel(QObject *parent = nullptr);
```

Comment:
This legacy name is inappropriate. It should be "TorrentListModel".

Reply:
Although, since all related classes use "TransferList" term, it should be TransferListModel.